daniel-alejandro-garcia/tarea-3/js/smallEnough.js

```diff
@@ -0,0 +1,11 @@
+function smallEnough(a, limit){
+    let bool;
+        for(let i = 0; i < a.length; i++){
```

Comment:
@da-garcia ,  use modern approaches for array operations like forEach, map, reduce.